src/database/seeders/20190724090406-user.js

```diff
@@ -1,11 +1,9 @@
-import bcrypt from 'bcrypt';
-
 export const up = queryInterface => queryInterface.bulkInsert('Users', [{
   firstName: 'Eden',
   lastName: 'Hazard',
   email: 'eden@gmail.com',
   username: 'eden101',
-  password: bcrypt.hashSync('edenHazard', 10),
+  password: '$2y$12$WYGmXXP9ZU/HuBw3c7Jy6eECt7L3f6St67HzgP.OpfHPeMavsoDZa',
```

Comment:
I feel using bcrypt would be better than using the raw encryption

Reply:
Alright!

Reply:
Changes made, thanks.